### PR TITLE
Update testcontainers to 1.19.5

### PR DESCRIPTION
**Pull request by KyleAure**
Planned:
- Update testcontainers to 1.19.5
- Update Oracle tests and db rotation to use Oracle 23c
- Update Oracle tests and db rotation to use oracle-free testcontaienrs (rebrand of oracle-xe)
- Update projects to use apache common-compress internal bundle for easier future updates

Found during testing:
- Update Ryuk version to 0.6.0
- Update fattest.simplicity unittests to account for breaking changes to private API
- Handle tests that still require Oracle-XE

### CODE REVIEW

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
Combined build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_nTzzMMnyEe677dsQcqWsjQ

Combined DB Rotation Build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_DK514MnzEe677dsQcqWsjQ

Target locations of links might be accessible only to IBM employees.

**Review by jmstephensgit — DISMISSED** [review]

**Comment by KyleAure** [comment]
#libby

**Comment by KyleAure** [comment]
Rebuilding with new commits

Combined build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_UraG4MqxEe677dsQcqWsjQ

Combined DB Rotation Build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_YjrY8MqxEe677dsQcqWsjQ

Target locations of links might be accessible only to IBM employees.

**Comment by KyleAure** [comment]
Rebuilding with new commits

Combined build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_FybBsMtnEe677dsQcqWsjQ

Combined DB Rotation Build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_BoEhIMtnEe677dsQcqWsjQ

Target locations of links might be accessible only to IBM employees.

**Comment by KyleAure** [comment]
🤞🏼 Last rebuild with new commits and no repository outages

Combined build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_hNDz0MwVEe677dsQcqWsjQ

Combined DB Rotation Build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_nWKgEMwVEe677dsQcqWsjQ

Target locations of links might be accessible only to IBM employees.

**Comment by KyleAure** [comment]
Build results:

- Combined build: Failed with a single known defect
- Combined DB Rotation Build: Failed three test buckets that are known to fail on Oracle [com.ibm.ws.jpa.tests.jpa_31_fat, io.openliberty.data.internal_fat, io.openliberty.data.internal_fat_jpa]

This is ready to be re-reviewed and merged #libby

**Review by jmstephensgit — DISMISSED** [review]

File: dev/fattest.simplicity/src/componenttest/topology/database/container/OracleXEContainer.java [hunk 1]
```diff
@@ -0,0 +1,214 @@
+package componenttest.topology.database.container;
```
Comment (tevans78) [inline dev/fattest.simplicity/src/componenttest/topology/database/container/OracleXEContainer.java:10]: Missing copyright header. Even if the original source didn't have one you should try to add something appropriate. Also, make it very clear where the original came from (github URL?) and what your changes are.

**Comment by KyleAure** [comment]
Fixed copyright headers and made some more updates.  Let's do another set of builds. #libby

Combined build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_RrHLMM0NEe677dsQcqWsjQ

Combined DB Rotation Build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_VW2dIM0NEe677dsQcqWsjQ

Target locations of links might be accessible only to IBM employees.

**Comment by KyleAure** [comment]
Previous builds failed. #libby

Combined build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_VTi1oM8qEe6-zvLIpypNYg

Combined DB Rotation Build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_Xiv4IM8qEe6-zvLIpypNYg

Target locations of links might be accessible only to IBM employees.

**Comment by KyleAure** [comment]
Previous builds failed. #libby

Combined build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_p_WEwM92Ee6-zvLIpypNYg

Combined DB Rotation Build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_n-ckAM92Ee6-zvLIpypNYg

Target locations of links might be accessible only to IBM employees.

**Comment by KyleAure** [comment]
Build results: 

Combined build: Successful
Combined DB Rotation Build: Test failures where expected.

Ready for review and merging.

**Review by jmstephensgit — APPROVED** [review]

**Pull request merged**